tox-uv.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
```

Comment:
It would be quite nicer if we could avoid a separate tox file. I figure tox-uv can be installed in CI. The other difference is `uv_resolution` and `uv_python_preference`. Is there a way to set them on the existing tox.ini file, preferably also controlled by CI envvars?

Reply:
Done